app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ActiveRecord::Base
+  has_many :posts
```

Comment:
- [ ] Great job with the associations and methods. please ensure they are set in accordance to this ERD diagram. kindly add the foreign key which is the `author id` when the user has many post. 
`has_many :posts, foreign_key: 'author_id'`